ports/stm32/boards/plli2svalues.py

```diff
@@ -137,8 +137,11 @@ def search_header(filename, re_include, re_define, lookup, val):
             m = regex_define.match(line)
             if m:
                 # found lookup value
+                found = m.group(3)
+                if "*" in found or "/" in found:
+                    found = eval(found)
```

Comment:
Did you find cases where this arithmetic was needed?  Can you give an example of a board that needs it?

Reply:
This logic is to support usage like in this PR for https://github.com/micropython/micropython/blob/aecdfc198723e45cf0a88baf8970f361a39f4014/ports/stm32/boards/WEACT_F411_BLACKPILL/stm32f4xx_hal_conf.h#L11
Where the clock speed define is reused in two locations, one location needing it is Hz the other needing it in MHz
https://github.com/micropython/micropython/blob/aecdfc198723e45cf0a88baf8970f361a39f4014/ports/stm32/boards/WEACT_F411_BLACKPILL/mpconfigboard.h#L23

Reply:
OK, thanks for that.  We should keep this bit of Python logic then.

Reply:
Thanks but yes I'll add a comment to the section to clarify its purpose